crates/cheatnet/tests/cheatcodes/load.rs

```diff
@@ -130,15 +126,13 @@ fn load_state_map_complex_value() {
     )
     .unwrap();
 
-    // This can be abstracted in cairo code itself, without compromising load's interface simplicity
```

Comment:
Is this comment no longer valid?

Reply:
Yes i think it's been abstracted pretty well in the `calculate_address` so that in cairo you just provide the `selector!` (which you provide yourself as a string) and the `Key`s will be taken from the appropriate hash-serializable key value (if the value you want to fetch is in a mapping you have to implement hash)

Reply:
Okay, so I guess a few lines down there is the same comment that is still present

Reply:
Deleted